mirgecom/initializers.py

```diff
@@ -9,6 +9,7 @@
 .. autoclass:: MulticomponentLump
 .. autoclass:: Uniform
 .. autoclass:: AcousticPulse
+.. autoclass:: MixtureInitializer
 .. automethod: _make_pulse
 .. automethod: _make_uniform_flow
```

Comment:
Sorry, unrelated, but only seeing this now: underscored functions should by definition not be part of a documented interface. Either un-underscore or remove from the docs.

Reply:
#230

Reply:
Yes! #231